health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -170,58 +176,77 @@
+    }
 
-        // Set the fill color to green for cell J1
-        worksheet.getCell("J1").fill = {
-            type: 'pattern',
-            pattern: 'solid',
-            fgColor: { argb: 'ff9248' } // Green color
+    // Populate username and password columns if data is provided
+    if (userNameAndPassword) {
+        // Set headers with formatting
+        const setCellHeader = (cell: string) => {
+            worksheet.getCell(cell).value = cell === usernameColumn + "1" ? "UserName" : "Password";
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)